jetstream/pipelines.py

```diff
@@ -163,7 +163,7 @@ def find_pipelines(*dirs):
                     p = Pipeline(path)
                     log.debug(f'Found {p} at {path}')
                     yield p
-                except :
```

Comment:
Looks like this would resolve errors similar to the following:
```
Exception ignored in: <generator object find_pipelines at 0x2aaac4eccac0>
    Traceback (most recent call last):
      File "jetstream/pipelines.py", line 194, in get_pipeline
        return p
    RuntimeError: generator ignored GeneratorExit
```

Reply:
Yep, that should be fixed now